src/nvim/misc1.c

```diff
@@ -2479,7 +2479,7 @@ int prompt_for_number(int *mouse_used)
   save_cmdline_row = cmdline_row;
   cmdline_row = 0;
   save_State = State;
-  State = CMDLINE;
+  State = ASKMORE;  // prevents a screen update when using a timer
```

Comment:
Not sure if that comment is actually true in our case but it probably doesn't matter.

Reply:
https://github.com/vim/vim/commit/4cbdf155ca7bb6fbdab2d5acafae5fc712fca8b8#diff-9cce1caa6d902c1868def5db926f2e01R3756
I guess not.